src/sdk/namespace_nb.js

```diff
@@ -64,6 +64,8 @@ class NamespaceNB {
 
     read_object_md(params, object_sdk) {
         if (this.target_bucket) params = _.defaults({ bucket: this.target_bucket }, params);
+        // Noobaa bucket does not currrently support partNumber query parameter. Ignore it for now.
+        _.unset(params, 'part_number');
```

Comment:
ok

Reply:
Actually setting params.part_number = undefined will cause invalid schema RPC error since the current schema validation treats undefined as that parameter is set. I guess we would have to use _,unset() @guymguym, dince part_number is set to undefined in s3_get_object.js if no partNumber query parameter. If you are still against calling _.unset(), perhaps because it is always called in namespace_nb.js, we can change s3_get_object.js to only set part_number when partNumber query is set. Made the suggest changes.

Reply:
ok